{{cookiecutter.project_slug}}/setup.py

```diff
@@ -14,7 +14,11 @@
 
 setup_requirements = [{%- if cookiecutter.use_pytest == 'y' %}'pytest-runner',{%- endif %} ]
 
-test_requirements = [{%- if cookiecutter.use_pytest == 'y' %}'pytest>=3',{%- endif %} ]
+test_requirements = [{%- if cookiecutter.use_pytest == 'y' %}'pytest>=3',{%- endif %}]
```

Comment:
all changes in this `setup.py` looks pretty generic, probable upstream would want it.

Reply:
Maybe!